src/interfaces/stepper-interfaces/YourCompany/address-interface.ts

```diff
@@ -18,8 +18,8 @@ export interface BaseAddressObjIF {
   deliveryAddress?: AddressIF
 }
 
-/** Interface to define the incorporation addresses. */
-export interface IncorporationAddressIf {
+/** Interface to define the addresses. */
+export interface AddressesIF {
```

Comment:
This name is probably OK for now but this is not a univeral address interface. For example, [registration business addresses](https://github.com/bcgov/business-create-ui/blob/main/src/interfaces/Registration/business-address-interface.ts) don't look like this.

If a better name comes to you, consider changing it 😁

Reply:
Alas, naming is hard. 
It does sound pretty universal but within the context of this app we have defined it at least. 
I would have preferred another format (ie a new interface to match an updated addresses responses from the api) but it sounds like there will be no changes on that end.

Reply:
Yes, naming is hard. (especially when you're old like me)

Actually, I think the API data (schema) is different for business addresses and other addresses. (Registered/Records office vs none)

nvm for now